main.js

```diff
@@ -13,7 +14,11 @@ function initApp() {
     function createWindow() {
         if (win === null) {
             const iconPath = path.join(__dirname, 'resources', 'icon.png');
-            const htmlUrl = `file://${__dirname}/index.html`;
+            const htmlUrl = url.format({
+                pathname: path.join(__dirname, 'index.html'),
+                protocol: 'file:',
+                slashes: true
```

Comment:
Is slashes needed?

Reply:
Not necessary, but https://electron.atom.io/docs/tutorial/quick-start/ suggests this as default way to contruct URL, although looking at [`url.format`](https://nodejs.org/api/url.html#url_url_format_urlobject) documentation, I don't think we need `//` appended to the end result, I can remove it. :+1:

Reply:
please, do it, everything else looks good